integration/mountkexec_test.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+package integration
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	"github.com/u-root/u-root/pkg/cp"
+	"github.com/u-root/u-root/pkg/golang"
+	"github.com/u-root/u-root/pkg/uroot"
+	"github.com/u-root/u-root/pkg/uroot/builder"
+	"github.com/u-root/u-root/pkg/uroot/initramfs"
+)
+
+// Generates an iso containing a kernel and initramfs to be used for testing mount and kexec
+func genISO() (err error) {
+	// Copy kernel to tmpDir for tests involving kexec.
+	tmpDir, err := ioutil.TempDir("", "uroot-integration")
+	if err != nil {
+		return err
+	}
+	kernel := filepath.Join(tmpDir, "kernel")
+	if err := cp.Copy(os.Getenv("UROOT_KERNEL"), kernel); err != nil {
+		return err
+	}
+
+	// OutputFile
+	newOutputFile := filepath.Join(tmpDir, "initiso.cpio")
+	w, err := initramfs.CPIO.OpenWriter(log.New(os.Stderr, "", 0), newOutputFile, "", "")
```

Comment:
Hey Ron.I don't understand what I am supposed to be doing here.First of all,I don't think I can use t.Log here because genISO doesn't take in a testing parameter,so I don't have access to t.Log.Also why should we never use log?

Reply:
damn github, I was seeing an older version, even after a refresh.

Anyway, I'm gonna let you work this one out. The problem: can't use log.New() here. So, you need to see how to fix it.

Reply:
did this get changed?

Reply:
ah, did you want tlogger here too?